doc/interop-test-descriptions.md

```diff
@@ -509,21 +508,29 @@ Procedure:
       }
     }
     ```
+The client attaches custom metadata with the following keys and values:
+    ```
+    key: "x-grpc-test-echo-initial", value: "test_initial_metadata_value"
+    key: "x-grpc-test-echo-trailing-bin", value: 0xababab
+    ```
+ 2. Client repeats step 1. with FullDuplexCall instead of UnaryCall.
 
 Asserts:
 * call was successful
-* custom metadata is echoed back in the response header.
-* custom metadata is echoed back in the response trailer.
+* metadata with key `"x-grpc-test-echo-initial"` and value `"test_initial_metadata_value"`is received in the initial metadata.
+* metadata with key `"x-grpc-test-echo-trailing-bin"` and value `0xababab` is received in the trailing metadata.
```

Comment:
This won't work, because the server-side Echo Metadata feature echos x-grpc-test-echo-trailing, not x-grpc-test-echo-trailing-bin. The server feature needs to be updated in some way. For example, if it also echoed x-grpc-test-echo-trailing-bin and x-grpc-test-echo-trailing-bin then it would work.